pkg/controller/zookeeper/zookeeper_controller.go

```diff
@@ -167,13 +171,8 @@ func (r *ReconcileZookeeper) Reconcile(request reconcile.Request) (reconcile.Res
 		return reconcile.Result{}, nil
 	}
 
-	currentZookeeperInstance := *instance
-
-	configMap, err := instance.CreateConfigMap(request.Name+"-"+instanceType+"-configmap", r.Client, r.Scheme, request)
-	if err != nil {
-		return reconcile.Result{}, err
-	}
-	configMap2, err := instance.CreateConfigMap(request.Name+"-"+instanceType+"-configmap-1", r.Client, r.Scheme, request)
+	configMapName := instance.Name + "-zookeeper-conf"
+	_, err := instance.CreateConfigMap(configMapName, r.Client, r.Scheme, request)
 	if err != nil {
```

Comment:
Nit: We can combine this err check with previous statement.

Reply:
fixed